ghcide/src/Development/IDE/Types/Exports.hs

```diff
@@ -158,49 +183,44 @@ type WithHieDb = forall a. (HieDb -> IO a) -> IO a
 createExportsMapHieDb :: WithHieDb -> IO ExportsMap
 createExportsMapHieDb withHieDb = do
     mods <- withHieDb getAllIndexedMods
-    idents <- forM (filter (nonInternalModules . modInfoName . hieModInfo) mods) $ \m -> do
+    idents' <- forM (filter (nonInternalModules . modInfoName . hieModInfo) mods) $ \m -> do
         let mn = modInfoName $ hieModInfo m
-            mText = pack $ moduleNameString mn
-        fmap (wrap . unwrap mText) <$> withHieDb (\hieDb -> getExportsForModule hieDb mn)
-    let exportsMap = Map.fromListWith (<>) (concat idents)
-    return $ ExportsMap exportsMap $ buildModuleExportMap (concat idents)
```

Comment:
This fixes a bug where the module exports map created from hiedb would be keyed by the identifier name instead of the module name. Caught because of using proper types instead of Text!